dev/TabView/TabView.xaml

```diff
@@ -0,0 +1,430 @@
+                                                                Background="Transparent"
+                                                                BorderThickness="1"
+                                                                Delay="50"
+                                                                FontFamily="Segoe MDL2 Assets"
+                                                                Interval="100"
+                                                                Style="{StaticResource RepeatButtonRevealStyle}"
+                                                                Visibility="Collapsed"
+                                                                Content="&#xE76B;"/>
+
+                                                            <ScrollContentPresenter x:Name="ScrollContentPresenter"
+                                                                Grid.Column="1"
+                                                                Padding="{TemplateBinding Padding}" />
+
+                                                            <RepeatButton x:Name="ScrollIncreaseButton"
+                                                                Grid.Column="2"
+                                                                VerticalAlignment="Bottom"
+                                                                HorizontalAlignment="Center"
+                                                                Width="{ThemeResource TabViewItemScrollButtonWidth}"
+                                                                Height="{ThemeResource TabViewItemScrollButtonHeight}"
+                                                                Padding="{ThemeResource TabViewItemScrollButtonPadding}"
+                                                                Background="Transparent"
+                                                                BorderThickness="1"
+                                                                Delay="50"
+                                                                FontFamily="Segoe MDL2 Assets"
+                                                                Interval="100"
+                                                                Style="{StaticResource RepeatButtonRevealStyle}"
+                                                                Visibility="Collapsed"
+                                                                Content="&#xE76C;"/>
+                                                        </Grid>
+                                                    </Border>
+                                                </ControlTemplate>
+                                            </Setter.Value>
+                                        </Setter>
+                                    </Style>
+                                </ScrollViewer.Style>
+
+                                <ItemsPresenter x:Name="TabsItemsPresenter"
```

Comment:
Just FYI for the future, we were thinking it could make sense to break out the layout logic for tabs into its own Panel to help account for the variable tab sizes as well as being able to layout the other types of content and padding around it to get things to be 'sticky' next to the tabs like we have in the toolkit or allow for customization of the padded area between the tabs/sticky content and the right content.  (And to help with drag and drop, as there's a clipping issue in the current toolkit control as we don't resize the area of the tabs when dragging a new tab in.

Reply:
Having its own panel would also enable the scrolling of tabs when there are many of them.  In the same way you get FlipView arrows on pivot headers.